pyqt-apps/siriushla/as_ap_sofb/graphics/base.py

```diff
@@ -19,14 +19,15 @@
 
 class BaseWidget(QWidget):
     """."""
-    DEFAULT_DIR = _pathlib.Path.home().as_posix()
-    EXT = '.txt'
-    EXT_FLT = 'Text Files (*.txt)'
+
+    DEFAULT_DIR = '/home/sirius/screens-iocs'
```

Comment:
The official (managed by ansible) path is `/home/sirius/mounts/screens-iocs`

Reply:
done!